glean-core/metrics.yaml

```diff
@@ -25,24 +25,6 @@ glean.baseline:
     notification_emails:
       - glean-team@mozilla.com
     expires: never
-  locale:
```

Comment:
You should check with @fbertsch if it's fine for us to remove this. Or maybe @mdboom knows as well!

I believe it should be fine.

Reply:
Yeah -- it should be fine.  The column will remain in the table (since it exists in historical revisions) so old versions will continue to successfully record it, but new ones will now.